formatTest/unit_tests/input/extensions.re

```diff
@@ -348,7 +348,7 @@ let () = {
   something_else();
 };
 
-[%bs.raw x => x];
```

Comment:
don't we also accept this form? I think it should keep being tested.

Reply:
Pushed back the test case, we do support them.

Reply:
It's still removed. did you forget to push?

Reply:
I added all of the cases in the cram tests. Let me add them in the `formatTest` as well